presbydora.module

```diff
@@ -273,9 +285,34 @@ function presbydora_embargo_date_set_year_range($form_element, $form_values) {
  */
 function presbydora_islandora_datastream_access($op, $islandora_datastream, $user) {
   if ($islandora_datastream->id == 'OBJ' || $islandora_datastream->id == 'PDF') {
+    // Modified to allow specific objects to be accessible.
+    $object = $islandora_datastream->parent;
+    $pid = $object->id;
+
+    module_load_include('inc', 'presbydora', 'includes/utilities');
+    $bypass_pids = variable_get('presbydora_access_bypass_pids', array());
+    if (!empty($bypass_pids)) {
+      $ancestors = presbydora_get_ancestors($pid);
+      $is_intersect = !empty(array_intersect($ancestors, $bypass_pids));
+
+      if ($is_intersect) {
+        return NULL;
+      }
+    }
     $guest = user_role_load_by_name('PHS Guest');
```

Comment:
One... kind of weird thing: We have logic around this, but don't have the role created in the env, in a feature or whatever... is fine for _this_ PR, but... makes testing it odd.

Reply:
Actually... whoops... oversight: It has `$user->uid == 0` there, for anonymous users as well, derp.